src/scripts/main.js

```diff
@@ -1,3 +1,44 @@
 'use strict';
 
-// write code here
+function sortRowsByNumber(rows, childSelector) {
+  let result = [];
+  const childNomer = childSelector + 1;
+
+  if (childSelector === 0 || childSelector === 1) {
+    result = [...rows].sort((a, b) => {
+      const lettersA = a.querySelector(
+        `td:nth-child(${childNomer})`,
+      ).textContent;
+      const lettersB = b.querySelector(
+        `td:nth-child(${childNomer})`,
+      ).textContent;
+
+      return lettersB.localeCompare(lettersA);
```

Comment:
The use of 'localeCompare' is correct for string comparison, but ensure that the text content in these columns is always intended to be sorted in descending order.